test/Interop/C/macros/redefined-macro.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=RedefinedMacro -I %S/Inputs -source-filename=x | %FileCheck %s
+
+// We cannot import redefined macros.
```

Comment:
Consider adding some positive tests, testing that we import valid macros.

Reply:
Will do.